languoids/tree/araw1281/nort2990/negr1239/wapi1252/mapi1252/md.ini

```diff
@@ -4,7 +4,7 @@ name = Mapidian-Mawayana
 hid = mpw
 level = language
 iso639-3 = mpw
-latitude = 1.9145
+latitude = 1.9144999999999999
```

Comment:
For some reason these coords always come back in. Is there a way to fix this regression?